go/vt/vtgate/executor_dml_test.go

```diff
@@ -531,6 +648,85 @@ func TestInsertSharded(t *testing.T) {
 	}
 }
 
+func TestInsertShardedAutocommitLookup(t *testing.T) {
+
+	vschema := `
+{
+	"sharded": true,
+	"vindexes": {
+		"hash_index": {
+			"type": "hash"
+		},
+		"name_user_map": {
+			"type": "lookup_hash",
+			"owner": "user",
+			"params": {
+				"table": "name_user_map",
+				"from": "name",
+				"to": "user_id",
+				"autocommit_on_insert": "true"
```

Comment:
It seems weird to use a string boolean value "true" when json supports boolean natively...